packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -15,17 +15,23 @@ const INSERTER_COLOR_DARK_MODE = 'white';
 const INSERTER_SIDE_LENGTH = 12;
 const INSERTER_BORDER_SIZE = 1;
 const INSERTER_HOVER_OFFSET = 5;
-
+const CELL_MINIMUM_LENGTH = 20;
 const CELL_RESIZER_WIDTH = 4;
+const TABLE_RESIZER_LENGTH = 12;
 const HORIZONTAL_RESIZER_HTML =
     '<div style="position: fixed; cursor: row-resize; user-select: none"></div>';
 const VERTICAL_RESIZER_HTML =
     '<div style="position: fixed; cursor: col-resize; user-select: none"></div>';
+const TABLE_RESIZER_HTML_LTR =
+    '<div style="position: fixed; cursor: nw-resize; user-select: none; border: 1px solid #808080"></div>';
```

Comment:
Can the color #808080 work correctly in dark mode?

Reply:
yes

![Capture_dark](https://user-images.githubusercontent.com/4762468/116504747-e6157680-a86d-11eb-8cdd-3efb4b205614.JPG)